pkg/make_rpm.py

```diff
@@ -281,16 +266,39 @@ def Build(self, spec_file, out_file):
 
 
 def main(argv=()):
+  parser = argparse.ArgumentParser(
+      description='Helper for building rpm packages',
+      fromfile_prefix_chars='@')
+
+  parser.add_argument('--name', required=True,
+                      help='The name of the software being packaged.')
+  parser.add_argument('--version', required=True,
+                      help='The version of the software being packaged.')
+  parser.add_argument('--release', required=True,
+                      help='The release of the software being packaged.')
+  parser.add_argument(
+      '--arch',
+      help='The CPU architecture of the software being packaged.')
+  parser.add_argument('--spec_file', required=True,
+                      help='The file containing the RPM specification.')
+  parser.add_argument('--out_file', required=True,
+                      help='The destination to save the resulting RPM file to.')
+  parser.add_argument('--rpmbuild', help='Path to rpmbuild executable.')
+  parser.add_argument('--debug', action='store_true', default=False,
+                      help='Print debug messages.')
+  parser.add_argument('files', nargs='*')
+
+  options = parser.parse_args()
```

Comment:
This form uses `sys.argv`, in which case the `argv` param is unused. I'd change this to pass in `argv`. Then `main()` can in principle be called programmatically by another module.

Reply:
Done